.github/workflows/docusaurus.yml

```diff
@@ -0,0 +1,38 @@
+name: Deploy Docusaurus
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
So, do we want to run this on every push to main or on every release? 🤔

Reply:
Not on every push - only if there are changes to affecting files (`paths`), to always have an up-to-date documentation